docs/application/vstools-mac/overview.md

```diff
@@ -1,5 +1,8 @@
 # Visual Studio for Mac Extension for Tizen
 
+> [!NOTE]
+> There are no further distribution plan for Visual Studio for Mac Extension for Tizen after Oct. 29, 2021 release (version 1.6).
+
 Visual Studio for Mac Extension for Tizen is an extension, which enables you to develop Tizen .NET applications easily using various Tizen project templates with Visual Studio for Mac.

```

Comment:
Using ‘that’ instead of ‘which’ is more appropriate in this situation, since it is used before a restrictive clause. The comma was removed because it was unnecessary.
```suggestion
Visual Studio for Mac Extension for Tizen is an extension that enables you to develop Tizen .NET applications easily using various Tizen project templates with Visual Studio for Mac.

```